packages/trader/src/Modules/MT5/Containers/mt5-compare-accounts-modal.jsx

```diff
@@ -31,7 +32,9 @@ const MT5AttributeDescriber = ({ name, tooltip, counter }) => {
             <p className='mt5-attribute-describer' onClick={toggleModal}>
                 {name}
                 <MobileWrapper>
-                    <span className='counter'>{counter}</span>
+                    <Text weight='bold' line_height='x' size='xxxs' className='counter'>
```

Comment:
is there line height x?, can we name it something else? or is it a typo?

Reply:
yes, we have it on theme.scss 
` //Line Height
    --text-lh-l: 1.75;
    --text-lh-m: 1.5;
    --text-lh-s: 1.25;
    --text-lh-x: 1.4;`